lms/static/scripts/frontend_apps/components/LaunchErrorDialog.tsx

```diff
@@ -32,15 +31,13 @@ const vitalsourceBookshelfURL = 'https://bookshelf.vitalsource.com';
  * Render an error that prevents an LTI launch from completing successfully.
  *
  * This is rendered in a non-cancelable modal.
- *
- * @param {LaunchErrorDialogProps} props
  */
 export default function LaunchErrorDialog({
-  busy,
+  loading,
```

Comment:
I took the opportunity to rename `busy` to `loading` for consistency with other components, as suggested in https://github.com/hypothesis/lms/pull/5031#discussion_r1104945190

However, it is used to be passed to nested `ErrorModal`s, which also use `busy`. I will refactpr those next.